frontend/src/lib/components/DisplayOption.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import { selected_store } from './store.js';
```

Comment:
Can you create a folder called `stores`, put this file into there and rename it to store.ts? (since we use typescript across the entire project). `components` folder should contain only the svelte components. Can you add a new path alias to ` svelte.config.js` with ```$stores: path.resolve('./src/lib/stores'),```? Path alias is a way that we reference our folder in a much easier way. Then, you can do 
```
import { selected_store } from '$stores/store.js';
```

sorry if it sounds confusing, let me know if you have question.